src/components/Rate/Rate.jsx

```diff
@@ -71,6 +73,7 @@ Rate.defaultProps = {
   disabled: false,
   firstColor: '#ffb940',
   gutter: 8,
+  nowrap: false,
```

Comment:
NIT: 想像不太到會想要 wrap 的 use case，如果預設就 nowrap 會比較好嗎？

```suggestion
  nowrap: true,
```